config.yml

```diff
@@ -18,6 +18,7 @@ defaults:
     artifactTableName:            Artifacts
     taskGroupTableName:           TaskGroups
     taskGroupMemberTableName:     TaskGroupMembers
+    taskGroupSizeTableName:       TaskGroupSizes
```

Comment:
hmm... Maybe we can it something `taskGroupActiveSetTableName`  ?? Just brain storming...